### PR TITLE
Add ability to create traffic alarm more than for one NAT Gateway

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new: 

 - Add to module ability to create more alarm for more than one AWS NAT Gateway using `count`

### CODE REVIEW

**Comment by kobrikx** [comment]
[Monitoring: Add alert for high NAT gateway traffic](https://trello.com/c/mG2Egv4W/173-monitoring-add-alert-for-high-nat-gateway-traffic)

File: README.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ This module provides settings:
 ```hcl
 
 module "nat_gateway_traffic" {
-  source = "../../terraform/terraform-nat-gateway-traffic-alert"
+  source = "hazelops/nat-gateway-traffic-alert/aws"
```
Comment (igorkotof) [inline README.md:20]: @kobrikx what about to specify version of source?

Comment (kobrikx) [reply README.md:20]: add  `version = "~> 1.0" `

**Review by igorkotof — APPROVED** [review]
looks good

**Pull request merged**